### PR TITLE
🌱 Add make generate-hcloud-token and bump versions

### PR DESCRIPTION
**Pull request by kranurag7**
this commit add new make target `make generate-hcloud-token` and bumps the version of some components in the Makefile.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [x] squash commits
- [ ] include documentation
- [ ] add unit tests

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -338,6 +338,11 @@ delete-mgt-cluster-registry: $(CTLPTL) ## Deletes Kind-dev Cluster and the local
 	$(CTLPTL) delete cluster kind-$(INFRA_SHORT)
 	$(CTLPTL) delete registry $(INFRA_SHORT)-registry
 
+generate-hcloud-token:
+	@test -x ./hack/ci-e2e-get-token.sh && echo "[+] ./hack/ci-e2e-get-token.sh is available" || echo "[-] make sure you have ./hack/ci-e2e-get-token.sh available in executable format"
```
Comment (kranurag7) [inline Makefile:342]: - for ensuring that we have the file with executable permission.

Comment (kranurag7) [reply Makefile]: This is not required, error message from `make` is enough. 
```bash
# when file is missing 
bash: line 1: ./hack/ci-e2e-get-token.sh: No such file or directory
# when file is missing permissions
bash: line 1: ./hack/ci-e2e-get-token.sh: Permission denied
```

File: Makefile [hunk 2]
```diff
@@ -338,6 +338,10 @@ delete-mgt-cluster-registry: $(CTLPTL) ## Deletes Kind-dev Cluster and the local
 	$(CTLPTL) delete cluster kind-$(INFRA_SHORT)
 	$(CTLPTL) delete registry $(INFRA_SHORT)-registry
 
+generate-hcloud-token:
+	@test $${TPS_TOKEN?Please set environment variable}
```
Comment (guettli) [inline Makefile:342]: I would use `./hack/ensure-env-variables.sh` like in the places in the Makefile.

**Review by guettli — APPROVED** [review]
lgtm. Only one optional suggestion.

File: Makefile (hunk 2 above)
Comment (kranurag7) [reply Makefile]: @guettli I updated it to invoke `./hack/ensure-env-variables.sh` for checking env variables.

**Review by guettli — APPROVED** [review]

**Pull request merged**